server/src/app/routes.ts

```diff
@@ -5,6 +5,8 @@ import { categoryRouter } from "../modules/category/category.route"
 import { brandRouter } from "../modules/brand/brand.route"
 import { unitRouter } from "../modules/unit/unit.route"
 import { productRouter } from "../modules/product/product.route"
+import { customerRouter } from "../modules/customer/customer.route"
+import { sotckRouter } from "../modules/stock/stock.route"
```

Comment:
We can rename the sotckRouter to stockRouter

Reply:
done